### ISSUE TITLE
Deployment of the Application to a free Infrastructure

### ISSUE THREAD

**Issue by sn0wcat** [amosproj/amos2024ss01-xcelerator-demo-app#25]
## User story

1. As a user/product owner
2. I want / need to preview the current state of the application
3. So that we can discuss the current state at any time with the industry partner.

We need a CI/CD deployment to some free infrastructure (there is e.g. a github student developer pack) from the CI/CD pipeline. We should research what is available and make a decision where to deploy the app. (I could provide some environment, but I think it is more in the spirit of open source to use one which is available for free, so that it is not dependent on me :)) 

@shahraz1998 @lamara11 

## Acceptance criteria

- [x] developers present researched free deployment infrastructures to IP.
- [x] Collective decision is made on which infrastructure to use. 

## Definition of done (DoD)

- [x] Added only after week 5
- [x] The same for all features
- [x] Here goes the project specific part

## DoD general criteria

- [x] Feature has been fully implemented
- [x] Feature has been merged into the mainline
- [x] All acceptance criteria were met
- [x] Product owner approved features
- [x] All tests are passing
- [x] Developers agreed to release

**Comment by lamara11** [comment]
@developers please research the infrastructures until next IP Meeting (14.05.2024)

**Comment by sn0wcat** [comment]
- [AWS Free Tier](https://aws.amazon.com/free/?gclid=Cj0KCQjwxeyxBhC7ARIsAC7dS3-GMQJxXZ2rG35FHRywjziyPGbrR33P70iLim7Y5TCmF80e0MUi5fIaAt2eEALw_wcB&trk=9ab5159b-247d-4917-a0ec-ec01d1af6bf9&sc_channel=ps&ef_id=Cj0KCQjwxeyxBhC7ARIsAC7dS3-GMQJxXZ2rG35FHRywjziyPGbrR33P70iLim7Y5TCmF80e0MUi5fIaAt2eEALw_wcB:G:s&s_kwcid=AL!4422!3!645133561110!e!!g!!aws%20free%20tier!19579657595!152087369744&all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=*all&awsf.Free%20Tier%20Categories=*all)

**Comment by lamara11** [comment]
@HaruspexSan  To look into Azure 
@Hydraneut  To look into AWS
